terraform/projects/prom-ec2/paas-production/extra-prometheus-scrape-configs.yml.tpl

```diff
@@ -1,3 +1,19 @@
+- job_name: dcs-sandbox
+  scheme: https
+  honor_labels: true
+  honor_timestamps: true
+  metrics_path: '/federate'
+  params:
+    "match[]":
+    # fetch everything (via https://stackoverflow.com/a/39253848 )
+    - '{__name__=~".+"}'
+  static_configs:
+  - targets:
+    - dcs-sandbox-prometheus.cloudapps.digital
```

Comment:
i made this domain up, i don't know what the correct thing is

Reply:
All domains are made up, man.